### DISCUSSION TITLE
Welcome to storyboarder Discussions!

### DISCUSSION THREAD

**Discussion by setpixel** [wonderunit/storyboarder#2176] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by knmurphy** [comment]
Howdy Charles and company! 

I'm Kevin and I'm pretty excited to be just starting out with Storyboarder in my tutorial video workflows. I'm currently working as a Developer Evangelist so my day-to-day involves developing instructional videos related to AWS. I've been producing video content for a little over 10 years though and this tool would have been so useful back when I was doing more narrative/story-driven on-location shoots. Happy to have it now though! 

I wanted to say THANK YOU! because at this point you've already built something really amazing and I'm enjoying using it! Cheers!

Going to share an idea from a media production community in a new post :-)

**Comment by TAOsame** [comment]
Howdy Charles and company!
I’m TAO, congratulations on the arrival of v3.0
I wish this tool can help story creators write more wonderful stories on the screen
Same as above
I want to say thank you! Because at this point you have already built something really amazing, so I like to use it! Cheers!

**Reply by TAOsame** [reply]
By the way, the idea of this community is nice

**Comment by ramAustralia** [comment]
Hi Charles,

I'm a "data wrangler" for a few script writers and directors/producers. 
Now that I found the source code here,  I'll try to compile it on the Linux
machines we use (running Pop!_OS Linux) and probably help out on some
aspects of code development.

I am also a long time member of SMPTE (Society of Motion Picture and
Television Engineers), AES (Audio Engineering Society), and elected to the
committee of the ASSG (Australian Screen Sound Guild).  I've built a
personal  LLMG (Large Language Model Generative) AI without guard rails
that certainly can generate some very interesting images, often NSFW.

My wife is a script writer and a member of WIFT (Woman in Film and
Television) and the AWG (Australian Writers Guild).

Regards,

Dr. Richard Anton Marschall
Chief Technical Officer
Marschall Acoustics Instruments Pty Limited
Level 26 (TEC)
One Bligh Street
Sydney, NSW  2000
Australia

[ram@hydrophones.com](https://www.hydrophones.com/webmail/src/compose.php?send_to=ram%40hydrophones.com)

websites:

http://www.hydrophones.com/
http://finevenuemusic.com/

**Comment by ramAustralia** [comment]
By the way, your wonderunit email address is routed through gmail which censors (blocks) most emails from outside the USA (at least).   Google is causing all kinds of problems for media organizations and environmental groups in Australia (where I am).  Please host some genuine email addresses on your wonderunit server(s).  Gmail is also impossible to reply to, well without extensive relaying.

**Comment by Mukhorin** [comment]
Hello, everybody!

I am an animator and architect from Ukraine. 
First of all, I would like to thank you for creating such an interesting and promising product.

I would like to share some feedback and suggestions on how the application could be improved. For people who draw, especially those using graphic tablets, it is critically important to have as much drawing space (canvas area) as possible. It is also essential to have smooth canvas zooming, panning and rotation, including support for touch gestures similar to those available on mobile devices.

To increase the available drawing area, I would suggest two possible approaches.

The first approach is to make the background around the marker toolbar at the top, the sketch thumbnails at the bottom, and the panel on the right transparent. This way, when the canvas is enlarged, it would no longer be clipped by those interface elements and could occupy almost the entire application window, while the tools and panels would simply overlay the canvas.

The second approach is to make the panels auto-hide when they are not in focus, leaving only a small strip or tab visible. Whenever a tool is needed, the user could move the mouse cursor or stylus over that strip or tab, and the panel would automatically expand into its full size.

I believe these changes would significantly improve the drawing experience and make the application more comfortable and efficient for artists and designers.

Best regards, Dmytro.
muhorin@ukr.net
https://mukhorin.github.io/
